### PR TITLE
[OSDOCS-2003]: AWS user-defined tags in machine sets

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.12+

Issue:
[OSDOCS-2003](https://issues.redhat.com//browse/OSDOCS-2003)

Link to docs preview:
- [Sample YAML for a compute machine set custom resource on AWS](https://53785--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machineset-yaml-aws_creating-machineset-aws) (regular compute machine)
- [Sample YAML for a compute machine set custom resource on AWS](https://53785--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#machineset-yaml-aws_creating-infrastructure-machinesets) (infrastructure machine)

(building...)

QE review:
- [ ] QE has approved this change.

Additional information:
- Cleans up AWS YAML sample condition use as well

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@sunzhaohua2 This dev work was done a long time ago, and is finally getting documented. Can you PTAL for QA verification?

File: modules/machineset-yaml-aws.adoc [hunk 1]
```diff
@@ -123,22 +122,29 @@ endif::infra[]
...
 endif::infra[]
 ifdef::infra[]
-<2> Specify the infrastructure ID, `<infra>` node label, and zone.
-<3> Specify the `<infra>` node label.
-<4> Specify a taint to prevent user workloads from being scheduled on infra nodes.
-<5> Specify a valid {op-system-first} AMI for your AWS zone for your {product-title} nodes. If you want to use an AWS Marketplace image, you must complete the {product-title} subscription from the link:https://aws.amazon.com/marketplace/fulfillment?productId=59ead7de-2540-4653-a8b0-fa7926d5c845[AWS Marketplace] to obtain an AMI ID for your region.
+<2> Specify the infrastructure ID, `infra` role node label, and zone.
+<3> Specify the `infra` role node label.
+endif::infra[]
+<4> Specify a valid {op-system-first} AMI for your AWS zone for your {product-title} nodes. If you want to use an AWS Marketplace image, you must complete the {product-title} subscription from the link:https://aws.amazon.com/marketplace/fulfillment?productId=59ead7de-2540-4653-a8b0-fa7926d5c845[AWS Marketplace] to obtain an AMI ID for your region.
```
Comment (kcarmichael08) [inline modules/machineset-yaml-aws.adoc:132]: spell out AMI at first use?

**Review by kcarmichael08 — COMMENTED** [review]
One question; otherwise, LGTM!

**Pull request merged**